e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -590,11 +590,11 @@ test.describe('Example Imagery in Time Strip', () => {
             name: 'Example Imagery'.concat(' ', uuid()),
             parent: timeStripObject.uuid
         });
+        // First navigate to the object you want to drag and drop
+        await page.goto(timeStripObject.url);
+        // Expand the tree to reveal the selected object
+        await page.click('button[title="Show selected item in tree"]');
```

Comment:
We only need to expand the tree if we're interacting with it. Since we're not, this line can be removed.

Reply:
Good catch, removed!